.travis.yml

```diff
@@ -36,7 +36,7 @@ before_install:
 install:
   - edm run -- python etstool.py install --runtime=${RUNTIME} || exit
   - git clone https://github.com/enthought/enthought-sphinx-theme.git
-  - pushd enthought-sphinx-theme && edm run -- python setup.py install && popd
+  - pushd enthought-sphinx-theme && edm run -- pip install -e . && popd
```

Comment:
It looks as though you're installing the theme into the bootstrap environment, when what you really want is to install it in the target EDM environment. The logic for that install should probably go into the `etstool.py` script, rather than directly into the `.travis.yml` config. (That way, it can be used by Appveyor, too.)